### DISCUSSION TITLE
Cannot save stories anymore

### DISCUSSION THREAD

**Discussion by Gibbon65** [fabd/kanji-koohii#321] [1. Feedback & Feature Requests]
When I go to options, study page when I try to save a story I get a note in the top story that says "No self vote!" and the story cannot be saved and then the kanji disappears.  Basically rendering the app useless for future study.  I am no kanji 784 and appears I cannot save anymore.

**Comment by fabd** [comment]
Hi Gibbon

To enter your own story you click in the middle of the white card at the top of the Study page rtight?

"No self vote" is when you clik the star below, but you don't need to do that your own story shows on top. Or, you can copy a story from soemone else with the small COPY icon next to the stars, and the story will also be shown at the top.
